src/renderer/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -158,7 +158,7 @@ const MnemonicConfirmScreen: React.FC<{ mnemonic: string; onConfirm: () => Promi
       </MnemonicStyled.TitleContainer>
       <MnemonicStyled.Form labelCol={{ span: 24 }} onFinish={handleFormSubmit}>
         <Form.Item name="mnemonic" validateStatus={mnemonicError && 'error'} help={!!mnemonicError && mnemonicError}>
-          <MnemonicPhrase words={sortedSelectedWords} onWordClick={handleRemoveWord} />
+          <MnemonicPhrase wordIcon={<DeleteOutlined />} words={sortedSelectedWords} onWordClick={handleRemoveWord} />
```

Comment:
![Screenshot from 2020-07-21 16-02-25](https://user-images.githubusercontent.com/61792675/88064577-c24d3380-cb6b-11ea-9277-a23c42c234b9.png)


Could you show the icon by hovering the button only, pls? It might confuse an user in some way.